src/CarbonAware/src/Model/EmissionsForecast.cs

```diff
@@ -17,21 +13,6 @@ public record EmissionsForecast
     /// Gets or sets the location the forecast is for.
     /// </summary>
     public Location Location { get; set; } = new();
-   
-    /// <summary>
-    /// Gets or sets the start time of the forecast data points.
-    /// </summary>
-    public DateTimeOffset DataStartAt { get; set; }
```

Comment:
Why did we remove start/end? Is it because it's not relevant to the forecast itself, just the query?

Reply:
correct! These are user request parameters which should be populated as is by the consumer DTOs.